superset/connectors/druid/models.py

```diff
@@ -146,53 +151,53 @@ def __html__(self):
         return self.__repr__()
 
     @property
-    def data(self):
+    def data(self) -> Dict:
         return {"id": self.id, "name": self.cluster_name, "backend": "druid"}
 
     @staticmethod
-    def get_base_url(host, port):
+    def get_base_url(host, port) -> str:
         if not re.match("http(s)?://", host):
             host = "http://" + host
 
         url = "{0}:{1}".format(host, port) if port else host
         return url
 
-    def get_base_broker_url(self):
+    def get_base_broker_url(self) -> str:
         base_url = self.get_base_url(self.broker_host, self.broker_port)
         return f"{base_url}/{self.broker_endpoint}"
 
-    def get_pydruid_client(self):
+    def get_pydruid_client(self) -> PyDruid:
         cli = PyDruid(
             self.get_base_url(self.broker_host, self.broker_port), self.broker_endpoint
         )
         if self.broker_user and self.broker_pass:
             cli.set_basic_auth_credentials(self.broker_user, self.broker_pass)
         return cli
 
-    def get_datasources(self):
+    def get_datasources(self) -> List[Dict]:
         endpoint = self.get_base_broker_url() + "/datasources"
         auth = requests.auth.HTTPBasicAuth(self.broker_user, self.broker_pass)
         return json.loads(requests.get(endpoint, auth=auth).text)
 
-    def get_druid_version(self):
+    def get_druid_version(self) -> str:
         endpoint = self.get_base_url(self.broker_host, self.broker_port) + "/status"
         auth = requests.auth.HTTPBasicAuth(self.broker_user, self.broker_pass)
         return json.loads(requests.get(endpoint, auth=auth).text)["version"]
 
-    @property
+    @property  # noqa
```

Comment:
what changed here requiring the `noqa` comment?

Reply:
My bad - I was using `mypy` instead of `flake8` to find typing errors, thinking they did the same thing. `mypy` seems to be a bit stricter, and it thought this line was an error even though it passed `flake8`, so I removed the unnecessary `# noqa`s that I had put in.

Reply:
I get this error: [Decorated property not supported](https://github.com/python/mypy/issues/1362).

The typechecker has a problem with decorated properties for some reason, and the typechecker only gets run on the file if there is at least one type hint used or something from `typing` is imported. So, what changed is that there is now typing in this file.

Reply:
Got it, can we add this comment instead then to only block certain linting and not all of it on this line? `# type: ignore`